tests/libtransmission/net-test.cc

```diff
@@ -180,8 +180,92 @@ TEST_F(NetTest, isGlobalUnicastAddress)
     }
 }
 
-TEST_F(NetTest, globalIPv6)
+namespace libtransmission::test
```

Comment:
the global ip cache tests should be moved into its own .cc file and added to the list of tests in CMakeLists.txt.

Happily, no xcode changes are necessary